src/components/modals/ExportRecordsModal.js

```diff
@@ -97,6 +97,7 @@ module.exports = {
           date: true,
           crypto: true,
           fiat: true,
+          vendorField: true,
```

Comment:
the stored options need to be merged with the default options, otherwise the vendorField toggle won't be visible, something like:

```
    exportOptions () {
      const defaultOptions = {
        delimiter: ',',
        includeHeaders: true,
        columns: {
          date: true,
          crypto: true,
          fiat: true,
          vendorField: true,
          id: true
        }
      }

      const storedOptions = walletApi.storage.get('exportOptions', true)

      if (!storedOptions) {
        return defaultOptions
      }

      const columns = {
        ...defaultOptions.columns,
        ...storedOptions.columns
      }

      return {
        ...defaultOptions,
        ...storedOptions,
        columns
      }
    },
```